web-registry/src/lib/db/types/json-ld/project-metadata.ts

```diff
@@ -1,60 +1,63 @@
-import { TypeValue, UrlList, UrlType } from 'lib/rdf/types';
+import { NameUrl, TypeValue, UrlList, UrlType } from 'lib/rdf/types';
 
+/** Anchored metadata AKA "Additional Info" - Editable only with a signed Ledger TX. */
 export interface ProjectMetadataLD {
   '@context': Context;
-  '@type': string;
+  '@type': string; // regen:C01-Project
   'schema:name': string;
-  'schema:image': UrlType;
-  'schema:creditText': string;
-  'schema:description'?: string;
   'schema:location': any;
   'regen:projectType': string;
-  'regen:projectActivity': ProjectActivity;
-  'regen:projectDeveloper': ProjectStakeholder;
-  'regen:landSteward': ProjectStakeholder;
-  'regen:landOwner': ProjectStakeholder;
-  'regen:projectOriginator': ProjectStakeholder;
+  'regen:projectActivity': NameUrl;
   'regen:offsetGenerationMethod': string;
   // TODO: not sure why regen:offsetGenerationMethod isn't compacting as expected so added this:
   'http://regen.network/offsetGenerationMethod'?: string;
   'regen:projectSize': ProjectSize;
   'regen:projectStartDate': TypeValue;
   'regen:projectEndDate': TypeValue;
-  'regen:boundaries': TypeValue;
-  'regen:creditClass': CreditClass;
-  'regen:landManagementActions': LandManagementActions;
+}
+
+/** Un-anchored metadata from our DB. This is editable without a Ledger TX. */
+export interface ProjectPageMetadataLD {
+  '@context': Context;
+  '@type': string; // regen:C01-Project-Page
+  'schema:image': UrlType;
+  'schema:creditText': string;
   'regen:galleryPhotos': UrlList;
   'regen:previewPhoto': UrlType;
   'regen:videoURL': UrlType;
+  'regen:projectDeveloper': ProjectStakeholder;
```

Comment:
as stated in https://github.com/regen-network/regen-web/pull/1701#discussion_r1049989145, we might just want to remove those roles from the json ld metadata and have them as project references

Reply:
As of latest convo with @wgwz, leaving these in anchored metadata